src/openloco/graphics/gfx.cpp

```diff
@@ -18,7 +21,7 @@ namespace openloco::gfx
     }
 
     static loco_global<drawpixelinfo_t, 0x0050B884> _screen_dpi;
-    static loco_global<g1_element[g1_expected_count::disc], 0x9E2424> _g1Elements;
+    static loco_global<g1_element[g1_expected_count::disc + 0x50000], 0x9E2424> _g1Elements;
```

Comment:
What is the `0x50000` representing? Is it to do with loaded object sprites? I think a new constant would be appropriate.